Makefile

```diff
@@ -130,7 +138,13 @@ chrome:
 		if [[ -z "$$(which chromium)" ]]; then \
 			make apt-get-chromium; \
 		fi; \
-		sudo ln -s "$$(which chromium)" /usr/bin/google-chrome; \
+		sudo ln -s "$$(which chromium)" $(CHROME_PATH); \
```

Comment:
This never runs on docker images destined for prod, does it? using `which chromium` to decide for us how to find a binary we need makes the security-worryer in me uneasy.

Reply:
Well, I agree this is ugly.